docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml

```diff
@@ -1,88 +1,212 @@
 name: 'Stanford HumSci'
 type: profile
 description: 'Installation profile for HumSci'
-version: 8.1.13
+version: 8.2.0
 core: 8.x
 themes:
   - material_admin
   - su_humsci_admin
   - stanford_basic
 dependencies:
-  - acquia_connector
-  - admin_toolbar
-  - admin_toolbar_links_access_filter
-  - admin_toolbar_tools
-  - block
-  - block_class
-  - block_content
-  - breakpoint
-  - chosen
-  - chosen_lib
-  - ckeditor
-  - color_field
-  - components
-  - config
-  - ctools
-  - datetime
-  - datetime_range
-  - dblog
-  - default_content
-  - dynamic_page_cache
-  - editor
-  - editor_advanced_link
-  - encrypt
-  - entity_reference_exposed_filters
-  - entity_reference_revisions
-  - environment_indicator
-  - field
-  - field_ui
-  - file
-  - filter
-  - help
-  - honeypot
-  - hs_basic_page
-  - hs_basic_page_display
-  - hs_text_area
-  - hs_text_area_display
-  - hs_hero_image_display
-  - image
-  - inline_form_errors
-  - key
-  - link
-  - linkit
-  - menu_admin_per_menu
-  - menu_block
-  - menu_link_content
-  - menu_ui
-  - metatag
-  - metatag_favicons
-  - nobots
-  - node
-  - options
-  - page_cache
-  - paragraphs
-  - path
-  - pathauto
-  - real_aes
-  - redirect
-  - redirect_404
-  - responsive_image
-  - role_delegation
-  - search
-  - shortcut
-  - simplesamlphp_auth
-  - stanford_media
-  - system
-  - taxonomy
-  - text
-  - token
-  - toolbar
-  - ui_patterns_ds
-  - ui_patterns_layouts
-  - ui_patterns_library
-  - ui_patterns_views
-  - update
-  - user
-  - views
-  - views_block_filter_block
-  - views_ui
+  - 'drupal:action'
```

Comment:
are `'` required?

Reply:
yeah i believe so since `:` is a yaml syntax character for objects/arrays.